datasets/google_dei/dataset.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dataset:
+  name: google_dei
+
+  friendly_name: ~
+  description: "Google Diversity, Equity, and Inclusion (DEI)"
+  dataset_sources: ~
+  terms_of_use: ~
+
+resources:
+  - type: bigquery_dataset
+    dataset_id: google_dei
+    description: "The diversity annual report from Google provides data on the representation, hiring, and retention of employees in our company including race and gender demographics."
+
+  - type: storage_bucket
+    name: "ggl-dei"
```

Comment:
I don't love this name but also don't have a better suggestion (I know, not a helpful comment)